### PR TITLE
- added typical household weight to FOOD database and BasicFood class

### PR DESCRIPTION
**Pull request by rgoolishian**
- Fix DB init script to clear existing database so always working with a clean db
- minor changes to tests broken by BasicFood constructor changes

### CODE REVIEW

**Comment by sparky005** [comment]
Looks good to me

**Comment by ghost** [comment]
me too. ready to merge I think.

File: src/main/java/edu/uml/diet/model/BasicFood.java [hunk 1]
```diff
@@ -160,6 +202,8 @@ public boolean equals(Object o) {
         if (fat != basicFood.fat) return false;
         if (protein != basicFood.protein) return false;
         if (!name.equals(basicFood.name)) return false;
+        if (householdWeight != basicFood.householdWeight) return false;
+        if (!householdWeightDescription.equals(basicFood.householdWeightDescription)) return false;
```
Comment (smarks) [inline src/main/java/edu/uml/diet/model/BasicFood.java:207]: even one line if statements should have braces. See Java coding conventions as to why http://www.oracle.com/technetwork/java/codeconventions-150003.pdf

**Comment by smarks** [comment]
looks ready to merge to me.

**Pull request closed without merging**